wiki-jpa-dao/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -51,11 +52,11 @@
         <constraints nullable="false"/>
       </column>
       <column name="OWNER" type="NVARCHAR(200)"/>
-      <column name="CREATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="CREATED_DATE" type="TIMESTAMP">
+        <constraints nullable="true"/>
       </column>
-      <column name="UPDATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="UPDATED_DATE" type="TIMESTAMP">
+        <constraints nullable="true"/>
```

Comment:
There isn't any impact here because we have a changeSet later to change all of these column to become NOT NULL with defaultValue is '1970-01-01 00:00:00'